hosting_docs/source/installation/2-manual-install.rst

```diff
@@ -163,23 +163,9 @@ our example cluster, the control machine is named “control1”.
 
         $ sudo update-alternatives --install /usr/bin/python python /usr/bin/python3 10
 
-5.  Now update pip; you might encounter installation issues otherwise.
```

Comment:
Jammy didn't like these steps; it printed out a warning about not running pip as root or you'll get permissions issues, and then later on there were permissions issues.